get_daily_news() run at :  2024-02-15 18:38:02.487930
get_daily_news() run at :  2024-02-15 18:40:02.509397
get_daily_news() run at :  2024-02-15 18:42:02.498155
get_daily_news() run at :  2024-02-15 18:44:02.474005
get_daily_news() run at :  2024-02-15 18:46:02.481460
get_daily_news() run at :  2024-02-15 18:48:02.477871
get_daily_news() run at :  2024-02-15 18:50:02.594578
get_daily_news() run at :  2024-02-15 18:52:02.644687
get_daily_news() run at :  2024-02-15 18:54:02.617861
get_daily_news() run at :  2024-02-15 18:56:02.577619
get_daily_news() run at :  2024-02-15 18:58:02.570113
get_daily_news() run at :  2024-02-15 19:00:02.567021
get_daily_news() run at :  2024-02-15 19:02:02.584238
get_daily_news() run at :  2024-02-15 19:04:02.612794
get_daily_news() run at :  2024-02-15 19:06:02.590357
get_daily_news() run at :  2024-02-15 19:08:02.549175
get_daily_news() run at :  2024-02-15 19:10:02.522234
Traceback (most recent call last):
  File "/home/zythrill/Django/portfolio/manage.py", line 22, in <module>
    main()
  File "/home/zythrill/Django/portfolio/manage.py", line 18, in main
    execute_from_command_line(sys.argv)
  File "/home/zythrill/Django/django_env/lib/python3.8/site-packages/django/core/management/__init__.py", line 442, in execute_from_command_line
    utility.execute()
  File "/home/zythrill/Django/django_env/lib/python3.8/site-packages/django/core/management/__init__.py", line 416, in execute
    django.setup()
  File "/home/zythrill/Django/django_env/lib/python3.8/site-packages/django/__init__.py", line 24, in setup
    apps.populate(settings.INSTALLED_APPS)
  File "/home/zythrill/Django/django_env/lib/python3.8/site-packages/django/apps/registry.py", line 116, in populate
    app_config.import_models()
  File "/home/zythrill/Django/django_env/lib/python3.8/site-packages/django/apps/config.py", line 269, in import_models
    self.models_module = import_module(models_module_name)
  File "/usr/lib/python3.8/importlib/__init__.py", line 127, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 1014, in _gcd_import
  File "<frozen importlib._bootstrap>", line 991, in _find_and_load
  File "<frozen importlib._bootstrap>", line 975, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 671, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 848, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/zythrill/Django/portfolio/news_analysis/models.py", line 5, in <module>
    class article(models.Model):
  File "/home/zythrill/Django/portfolio/news_analysis/models.py", line 11, in article
    date_recieved = models.DateTimeField(default=timezone.now)
NameError: name 'timezone' is not defined
Traceback (most recent call last):
  File "/home/zythrill/Django/portfolio/manage.py", line 22, in <module>
    main()
  File "/home/zythrill/Django/portfolio/manage.py", line 18, in main
    execute_from_command_line(sys.argv)
  File "/home/zythrill/Django/django_env/lib/python3.8/site-packages/django/core/management/__init__.py", line 442, in execute_from_command_line
    utility.execute()
  File "/home/zythrill/Django/django_env/lib/python3.8/site-packages/django/core/management/__init__.py", line 416, in execute
    django.setup()
  File "/home/zythrill/Django/django_env/lib/python3.8/site-packages/django/__init__.py", line 24, in setup
    apps.populate(settings.INSTALLED_APPS)
  File "/home/zythrill/Django/django_env/lib/python3.8/site-packages/django/apps/registry.py", line 116, in populate
    app_config.import_models()
  File "/home/zythrill/Django/django_env/lib/python3.8/site-packages/django/apps/config.py", line 269, in import_models
    self.models_module = import_module(models_module_name)
  File "/usr/lib/python3.8/importlib/__init__.py", line 127, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 1014, in _gcd_import
  File "<frozen importlib._bootstrap>", line 991, in _find_and_load
  File "<frozen importlib._bootstrap>", line 975, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 671, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 848, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/zythrill/Django/portfolio/news_analysis/models.py", line 5, in <module>
    class article(models.Model):
  File "/home/zythrill/Django/portfolio/news_analysis/models.py", line 11, in article
    date_recieved = models.DateTimeField(default=timezone.now)
NameError: name 'timezone' is not defined
Traceback (most recent call last):
  File "/home/zythrill/Django/portfolio/manage.py", line 22, in <module>
    main()
  File "/home/zythrill/Django/portfolio/manage.py", line 18, in main
    execute_from_command_line(sys.argv)
  File "/home/zythrill/Django/django_env/lib/python3.8/site-packages/django/core/management/__init__.py", line 442, in execute_from_command_line
    utility.execute()
  File "/home/zythrill/Django/django_env/lib/python3.8/site-packages/django/core/management/__init__.py", line 416, in execute
    django.setup()
  File "/home/zythrill/Django/django_env/lib/python3.8/site-packages/django/__init__.py", line 24, in setup
    apps.populate(settings.INSTALLED_APPS)
  File "/home/zythrill/Django/django_env/lib/python3.8/site-packages/django/apps/registry.py", line 116, in populate
    app_config.import_models()
  File "/home/zythrill/Django/django_env/lib/python3.8/site-packages/django/apps/config.py", line 269, in import_models
    self.models_module = import_module(models_module_name)
  File "/usr/lib/python3.8/importlib/__init__.py", line 127, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 1014, in _gcd_import
  File "<frozen importlib._bootstrap>", line 991, in _find_and_load
  File "<frozen importlib._bootstrap>", line 975, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 671, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 848, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/zythrill/Django/portfolio/news_analysis/models.py", line 5, in <module>
    class article(models.Model):
  File "/home/zythrill/Django/portfolio/news_analysis/models.py", line 9, in article
    date_added = models.DateTimeField(default=timezone.now)
NameError: name 'timezone' is not defined
get_daily_news() run at :  2024-02-15 19:18:01.909317
get_daily_news() run at :  2024-02-15 19:20:02.039828
get_daily_news() run at :  2024-02-15 19:22:02.133474
Traceback (most recent call last):
  File "/home/zythrill/Django/portfolio/manage.py", line 22, in <module>
    main()
  File "/home/zythrill/Django/portfolio/manage.py", line 18, in main
    execute_from_command_line(sys.argv)
  File "/home/zythrill/Django/django_env/lib/python3.8/site-packages/django/core/management/__init__.py", line 442, in execute_from_command_line
    utility.execute()
  File "/home/zythrill/Django/django_env/lib/python3.8/site-packages/django/core/management/__init__.py", line 416, in execute
    django.setup()
  File "/home/zythrill/Django/django_env/lib/python3.8/site-packages/django/__init__.py", line 24, in setup
    apps.populate(settings.INSTALLED_APPS)
  File "/home/zythrill/Django/django_env/lib/python3.8/site-packages/django/apps/registry.py", line 124, in populate
    app_config.ready()
  File "/home/zythrill/Django/django_env/lib/python3.8/site-packages/django/contrib/admin/apps.py", line 27, in ready
    self.module.autodiscover()
  File "/home/zythrill/Django/django_env/lib/python3.8/site-packages/django/contrib/admin/__init__.py", line 50, in autodiscover
    autodiscover_modules("admin", register_to=site)
  File "/home/zythrill/Django/django_env/lib/python3.8/site-packages/django/utils/module_loading.py", line 58, in autodiscover_modules
    import_module("%s.%s" % (app_config.name, module_to_search))
  File "/usr/lib/python3.8/importlib/__init__.py", line 127, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 1014, in _gcd_import
  File "<frozen importlib._bootstrap>", line 991, in _find_and_load
  File "<frozen importlib._bootstrap>", line 975, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 671, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 848, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/zythrill/Django/portfolio/news_analysis/admin.py", line 3, in <module>
    from .models import article
ImportError: cannot import name 'article' from 'news_analysis.models' (/home/zythrill/Django/portfolio/news_analysis/models.py)
get_daily_news() run at :  2024-02-15 19:26:02.628961
Failed to complete cronjob at ('*/2 * * * *', 'news_analysis.cron.get_daily_news', '>> /home/zythrill/Django/portfolio/log/cron.log 2>&1 ')
Traceback (most recent call last):
  File "/home/zythrill/Django/django_env/lib/python3.8/site-packages/django_crontab/crontab.py", line 145, in run_job
    func(*job_args, **job_kwargs)
  File "/home/zythrill/Django/portfolio/news_analysis/cron.py", line 29, in get_daily_news
    context['articles'] = article_context
NameError: name 'context' is not defined
{'status': 'ok', 'totalResults': 1031, 'articles': [{'source': {'id': None, 'name': 'Forbes'}, 'author': 'https://www.forbes.com/', 'title': 'Bitcoin Breaks $50K For First Time Since 2021', 'description': 'Steven Ehrlich, the director of Forbes Digital Assets, joins “Forbes Talks” to discuss Bitcoin reaching $50,000 for the first time since 2021.', 'url': 'https://www.forbes.com/video/6346755958112/bitcoin-breaks-50k-for-first-time-since-2021/', 'urlToImage': 'https://imageio.forbes.com/i-forbesimg/media/video/2024/02/13/6346755958112_still.jpg?format=jpg&height=460&width=800&fit=bounds', 'publishedAt': '2024-02-13T16:00:50Z', 'content': 'Steven Ehrlich, the director of Forbes Digital Assets, joins Forbes Talks to discuss Bitcoin reaching $50,000 for the first time since 2021.'}, {'source': {'id': None, 'name': 'Forbes'}, 'author': 'Abubakar Nur Khalil, Contributor, \n Abubakar Nur Khalil, Contributor\n https://www.forbes.com/sites/abubakarnurkhalil/', 'title': 'As Bitcoin ETFs Take Center Stage, Coinbase Diversifies', 'description': 'The exchange became an industry darling by offering a user-friendly product and promising to adhere to regulations. Now, new low-fee bitcoin funds are eating into its lucrative transaction fee business.', 'url': 'https://www.forbes.com/sites/digital-assets/2024/02/13/as-bitcoin-etfs-take-center-stage-coinbase-diversifies/', 'urlToImage': 'https://imageio.forbes.com/specials-images/imageserve/615daf3682a521d2906bb67d/0x0.jpg?format=jpg&crop=3301,1859,x0,y303,safe&height=900&width=1600&fit=bounds', 'publishedAt': '2024-02-13T15:01:25Z', 'content': 'Brian Armstrong, founder of Coinbase, photographed for Forbes by Jamel Toppin in January 2020.\r\nJamel Toppin/The Forbes Collection\r\nOverview\r\nCrypto exchange Coinbase has been an industry bellwether … [+9766 chars]'}, {'source': {'id': None, 'name': 'Forbes'}, 'author': 'Bill Sarubbi, Contributor, \n Bill Sarubbi, Contributor\n https://www.forbes.com/sites/greatspeculations/people/billsarubbi/', 'title': 'Stocks This Week: Buy Caterpillar And Coinbase', 'description': 'The market is overbought is certain industries. The industrial sector is now strengthening. Buy Caterpillar. There is also a stock that is likely to follow bitcoin up.', 'url': 'https://www.forbes.com/sites/greatspeculations/2024/02/12/stocks-this-week-buy-caterpillar-and-coinbase/', 'urlToImage': 'https://imageio.forbes.com/specials-images/imageserve/65c8cdfa37e9302b84fadc73/0x0.jpg?format=jpg&height=900&width=1600&fit=bounds', 'publishedAt': '2024-02-12T11:00:00Z', 'content': 'Photographer: Michael Nagle/Bloomberg\r\n© 2023 Bloomberg Finance LP\r\nThe next week is options expiration (OPEX) week which has been bearish in the month of February. This suggests a pullback next week… [+2798 chars]'}, {'source': {'id': None, 'name': 'Forbes'}, 'author': 'Forbes Daily, Forbes Staff, \n Forbes Daily, Forbes Staff\n https://www.forbes.com/sites/forbesdaily/', 'title': 'Forbes Daily: Las Vegas Is Officially A Sports Town', 'description': "Kansas City wins second straight Super Bowl; Nasdaq hits two-year high; Bitcoin rallies; Japan seeks to woo remote workers; what Disney's bet on Epic means for gaming.", 'url': 'https://www.forbes.com/sites/forbesdaily/2024/02/12/forbes-daily-las-vegas-is-officially-a-sports-town/', 'urlToImage': 'https://imageio.forbes.com/specials-images/imageserve/65ca20a284600cd7732bef2c/0x0.jpg?format=jpg&crop=3320,1866,x0,y171,safe&height=900&width=1600&fit=bounds', 'publishedAt': '2024-02-12T13:51:36Z', 'content': 'This is a published version of the Forbes Daily newsletter, you can sign-up to get Forbes Daily in your inbox here.\r\nGood morning,\r\nIn college, I double majored in journalism and political science. A… [+9587 chars]'}, {'source': {'id': None, 'name': 'Forbes'}, 'author': 'Danielle Chemtob, Forbes Staff, \n Danielle Chemtob, Forbes Staff\n https://www.forbes.com/sites/daniellechemtob/', 'title': 'Forbes Daily: Nvidia Passes Tech Giants Amazon And Alphabet', 'description': "Biden pushes for Israel Gaza ceasefire; Forbes Fintech 50; Bitcoin reaches $50,000 mark; Google to train people in AI; Samuel Alito's net worth; DC sports stadium snag.", 'url': 'https://www.forbes.com/sites/forbesdaily/2024/02/13/forbes-daily-nvidia-passes-tech-giants-amazon-and-alphabet/', 'urlToImage': 'https://imageio.forbes.com/specials-images/imageserve/65cb68f2969a440669b603b3/0x0.jpg?format=jpg&crop=1892,1065,x0,y97,safe&height=900&width=1600&fit=bounds', 'publishedAt': '2024-02-13T13:06:25Z', 'content': 'This is a published version of the Forbes Daily newsletter, you can sign-up to get Forbes Daily in your inbox here.\r\nGood morning,\r\nLike other artists, Usher wasnt paid for his Super Bowl halftime sh… [+11290 chars]'}]}
Failed to complete cronjob at ('*/2 * * * *', 'news_analysis.cron.get_daily_news', '>> /home/zythrill/Django/portfolio/log/cron.log 2>&1 ')
Traceback (most recent call last):
  File "/home/zythrill/Django/django_env/lib/python3.8/site-packages/django_crontab/crontab.py", line 145, in run_job
    func(*job_args, **job_kwargs)
  File "/home/zythrill/Django/portfolio/news_analysis/cron.py", line 29, in get_daily_news
    context['articles'] = article_context
NameError: name 'context' is not defined
{'status': 'ok', 'totalResults': 1031, 'articles': [{'source': {'id': None, 'name': 'Forbes'}, 'author': 'https://www.forbes.com/', 'title': 'Bitcoin Breaks $50K For First Time Since 2021', 'description': 'Steven Ehrlich, the director of Forbes Digital Assets, joins “Forbes Talks” to discuss Bitcoin reaching $50,000 for the first time since 2021.', 'url': 'https://www.forbes.com/video/6346755958112/bitcoin-breaks-50k-for-first-time-since-2021/', 'urlToImage': 'https://imageio.forbes.com/i-forbesimg/media/video/2024/02/13/6346755958112_still.jpg?format=jpg&height=460&width=800&fit=bounds', 'publishedAt': '2024-02-13T16:00:50Z', 'content': 'Steven Ehrlich, the director of Forbes Digital Assets, joins Forbes Talks to discuss Bitcoin reaching $50,000 for the first time since 2021.'}, {'source': {'id': None, 'name': 'Forbes'}, 'author': 'Abubakar Nur Khalil, Contributor, \n Abubakar Nur Khalil, Contributor\n https://www.forbes.com/sites/abubakarnurkhalil/', 'title': 'As Bitcoin ETFs Take Center Stage, Coinbase Diversifies', 'description': 'The exchange became an industry darling by offering a user-friendly product and promising to adhere to regulations. Now, new low-fee bitcoin funds are eating into its lucrative transaction fee business.', 'url': 'https://www.forbes.com/sites/digital-assets/2024/02/13/as-bitcoin-etfs-take-center-stage-coinbase-diversifies/', 'urlToImage': 'https://imageio.forbes.com/specials-images/imageserve/615daf3682a521d2906bb67d/0x0.jpg?format=jpg&crop=3301,1859,x0,y303,safe&height=900&width=1600&fit=bounds', 'publishedAt': '2024-02-13T15:01:25Z', 'content': 'Brian Armstrong, founder of Coinbase, photographed for Forbes by Jamel Toppin in January 2020.\r\nJamel Toppin/The Forbes Collection\r\nOverview\r\nCrypto exchange Coinbase has been an industry bellwether … [+9766 chars]'}, {'source': {'id': None, 'name': 'Forbes'}, 'author': 'Bill Sarubbi, Contributor, \n Bill Sarubbi, Contributor\n https://www.forbes.com/sites/greatspeculations/people/billsarubbi/', 'title': 'Stocks This Week: Buy Caterpillar And Coinbase', 'description': 'The market is overbought is certain industries. The industrial sector is now strengthening. Buy Caterpillar. There is also a stock that is likely to follow bitcoin up.', 'url': 'https://www.forbes.com/sites/greatspeculations/2024/02/12/stocks-this-week-buy-caterpillar-and-coinbase/', 'urlToImage': 'https://imageio.forbes.com/specials-images/imageserve/65c8cdfa37e9302b84fadc73/0x0.jpg?format=jpg&height=900&width=1600&fit=bounds', 'publishedAt': '2024-02-12T11:00:00Z', 'content': 'Photographer: Michael Nagle/Bloomberg\r\n© 2023 Bloomberg Finance LP\r\nThe next week is options expiration (OPEX) week which has been bearish in the month of February. This suggests a pullback next week… [+2798 chars]'}, {'source': {'id': None, 'name': 'Forbes'}, 'author': 'Forbes Daily, Forbes Staff, \n Forbes Daily, Forbes Staff\n https://www.forbes.com/sites/forbesdaily/', 'title': 'Forbes Daily: Las Vegas Is Officially A Sports Town', 'description': "Kansas City wins second straight Super Bowl; Nasdaq hits two-year high; Bitcoin rallies; Japan seeks to woo remote workers; what Disney's bet on Epic means for gaming.", 'url': 'https://www.forbes.com/sites/forbesdaily/2024/02/12/forbes-daily-las-vegas-is-officially-a-sports-town/', 'urlToImage': 'https://imageio.forbes.com/specials-images/imageserve/65ca20a284600cd7732bef2c/0x0.jpg?format=jpg&crop=3320,1866,x0,y171,safe&height=900&width=1600&fit=bounds', 'publishedAt': '2024-02-12T13:51:36Z', 'content': 'This is a published version of the Forbes Daily newsletter, you can sign-up to get Forbes Daily in your inbox here.\r\nGood morning,\r\nIn college, I double majored in journalism and political science. A… [+9587 chars]'}, {'source': {'id': None, 'name': 'Forbes'}, 'author': 'Danielle Chemtob, Forbes Staff, \n Danielle Chemtob, Forbes Staff\n https://www.forbes.com/sites/daniellechemtob/', 'title': 'Forbes Daily: Nvidia Passes Tech Giants Amazon And Alphabet', 'description': "Biden pushes for Israel Gaza ceasefire; Forbes Fintech 50; Bitcoin reaches $50,000 mark; Google to train people in AI; Samuel Alito's net worth; DC sports stadium snag.", 'url': 'https://www.forbes.com/sites/forbesdaily/2024/02/13/forbes-daily-nvidia-passes-tech-giants-amazon-and-alphabet/', 'urlToImage': 'https://imageio.forbes.com/specials-images/imageserve/65cb68f2969a440669b603b3/0x0.jpg?format=jpg&crop=1892,1065,x0,y97,safe&height=900&width=1600&fit=bounds', 'publishedAt': '2024-02-13T13:06:25Z', 'content': 'This is a published version of the Forbes Daily newsletter, you can sign-up to get Forbes Daily in your inbox here.\r\nGood morning,\r\nLike other artists, Usher wasnt paid for his Super Bowl halftime sh… [+11290 chars]'}]}
get_daily_news() run at :  2024-02-15 19:32:02.557272
get_daily_news() run at :  2024-02-15 19:34:02.681768
get_daily_news() run at :  2024-02-15 19:36:02.832137
get_daily_news() run at :  2024-02-15 19:38:02.937472
get_daily_news() run at :  2024-02-15 19:40:02.103240
get_daily_news() run at :  2024-02-15 19:42:02.297436
get_daily_news() run at :  2024-02-15 19:44:02.691161
get_daily_news() run at :  2024-02-15 19:46:02.989920
get_daily_news() run at :  2024-02-15 19:48:02.401810
get_daily_news() run at :  2024-02-15 19:50:02.575626
get_daily_news() run at :  2024-02-15 19:52:02.702097
get_daily_news() run at :  2024-02-15 19:54:02.839094
get_daily_news() run at :  2024-02-15 19:56:03.140562
get_daily_news() run at :  2024-02-15 19:58:02.305685
get_daily_news() run at :  2024-02-15 20:00:02.449996
get_daily_news() run at :  2024-02-15 20:02:02.638148
get_daily_news() run at :  2024-02-15 20:04:02.739069
get_daily_news() run at :  2024-02-15 20:06:02.907943
get_daily_news() run at :  2024-02-15 20:08:02.740250
get_daily_news() run at :  2024-02-15 20:10:03.152957
get_daily_news() run at :  2024-02-15 20:12:02.323530
get_daily_news() run at :  2024-02-15 20:14:02.568359
get_daily_news() run at :  2024-02-15 20:16:02.720745
get_daily_news() run at :  2024-02-15 20:18:02.946220
get_daily_news() run at :  2024-02-15 20:20:02.088512
get_daily_news() run at :  2024-02-15 20:22:02.293778
get_daily_news() run at :  2024-02-15 20:24:02.455039
get_daily_news() run at :  2024-02-15 20:26:02.641082
get_daily_news() run at :  2024-02-15 20:28:02.887730
get_daily_news() run at :  2024-02-15 20:30:02.059125
get_daily_news() run at :  2024-02-15 20:32:02.257328
get_daily_news() run at :  2024-02-15 20:34:02.540250
get_daily_news() run at :  2024-02-15 20:36:02.636757
get_daily_news() run at :  2024-02-15 20:38:02.952850
get_daily_news() run at :  2024-02-15 20:40:02.205635
get_daily_news() run at :  2024-02-15 20:42:02.351982
get_daily_news() run at :  2024-02-15 20:44:02.664003
get_daily_news() run at :  2024-02-15 20:46:02.812382
get_daily_news() run at :  2024-02-15 20:48:03.002917
get_daily_news() run at :  2024-02-15 20:50:02.331060
get_daily_news() run at :  2024-02-15 20:52:02.569906
Traceback (most recent call last):
  File "/home/zythrill/Django/portfolio/manage.py", line 22, in <module>
    main()
  File "/home/zythrill/Django/portfolio/manage.py", line 18, in main
    execute_from_command_line(sys.argv)
  File "/home/zythrill/Django/django_env/lib/python3.8/site-packages/django/core/management/__init__.py", line 442, in execute_from_command_line
    utility.execute()
  File "/home/zythrill/Django/django_env/lib/python3.8/site-packages/django/core/management/__init__.py", line 436, in execute
    self.fetch_command(subcommand).run_from_argv(self.argv)
  File "/home/zythrill/Django/django_env/lib/python3.8/site-packages/django/core/management/base.py", line 412, in run_from_argv
    self.execute(*args, **cmd_options)
  File "/home/zythrill/Django/django_env/lib/python3.8/site-packages/django/core/management/base.py", line 453, in execute
    self.check()
  File "/home/zythrill/Django/django_env/lib/python3.8/site-packages/django/core/management/base.py", line 485, in check
    all_issues = checks.run_checks(
  File "/home/zythrill/Django/django_env/lib/python3.8/site-packages/django/core/checks/registry.py", line 88, in run_checks
    new_errors = check(app_configs=app_configs, databases=databases)
  File "/home/zythrill/Django/django_env/lib/python3.8/site-packages/django/core/checks/urls.py", line 14, in check_url_config
    return check_resolver(resolver)
  File "/home/zythrill/Django/django_env/lib/python3.8/site-packages/django/core/checks/urls.py", line 24, in check_resolver
    return check_method()
  File "/home/zythrill/Django/django_env/lib/python3.8/site-packages/django/urls/resolvers.py", line 494, in check
    for pattern in self.url_patterns:
  File "/home/zythrill/Django/django_env/lib/python3.8/site-packages/django/utils/functional.py", line 57, in __get__
    res = instance.__dict__[self.name] = self.func(instance)
  File "/home/zythrill/Django/django_env/lib/python3.8/site-packages/django/urls/resolvers.py", line 715, in url_patterns
    patterns = getattr(self.urlconf_module, "urlpatterns", self.urlconf_module)
  File "/home/zythrill/Django/django_env/lib/python3.8/site-packages/django/utils/functional.py", line 57, in __get__
    res = instance.__dict__[self.name] = self.func(instance)
  File "/home/zythrill/Django/django_env/lib/python3.8/site-packages/django/urls/resolvers.py", line 708, in urlconf_module
    return import_module(self.urlconf_name)
  File "/usr/lib/python3.8/importlib/__init__.py", line 127, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 1014, in _gcd_import
  File "<frozen importlib._bootstrap>", line 991, in _find_and_load
  File "<frozen importlib._bootstrap>", line 975, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 671, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 848, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/zythrill/Django/portfolio/master_template/urls.py", line 25, in <module>
    path('news_analysis/', include('news_analysis.urls')),
  File "/home/zythrill/Django/django_env/lib/python3.8/site-packages/django/urls/conf.py", line 38, in include
    urlconf_module = import_module(urlconf_module)
  File "/usr/lib/python3.8/importlib/__init__.py", line 127, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 1014, in _gcd_import
  File "<frozen importlib._bootstrap>", line 991, in _find_and_load
  File "<frozen importlib._bootstrap>", line 975, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 671, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 848, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/zythrill/Django/portfolio/news_analysis/urls.py", line 2, in <module>
    from . import views
  File "/home/zythrill/Django/portfolio/news_analysis/views.py", line 29
    pos++
        ^
SyntaxError: invalid syntax
get_daily_news() run at :  2024-02-15 20:56:02.429704
get_daily_news() run at :  2024-02-15 20:58:02.695163
get_daily_news() run at :  2024-02-15 21:00:02.877366
get_daily_news() run at :  2024-02-15 21:02:02.117681
get_daily_news() run at :  2024-02-15 21:04:02.467777
get_daily_news() run at :  2024-02-15 21:06:02.726874
get_daily_news() run at :  2024-02-15 21:08:03.127925
get_daily_news() run at :  2024-02-15 21:10:02.419954
get_daily_news() run at :  2024-02-15 21:12:02.540675
get_daily_news() run at :  2024-02-15 21:14:02.754449
get_daily_news() run at :  2024-02-15 21:16:03.062312
get_daily_news() run at :  2024-02-15 21:18:02.549376
get_daily_news() run at :  2024-02-15 21:20:02.833274
